catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -273,8 +273,13 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
     }
 
     // add in all the solr query metrics to return
-    metrics.put(SQCMB + "solrquery" + QM_ELAPSED, System.nanoTime() - startQuery);
+    Long prevSolrTime = (Long) responseProps.get(SQCMB + "solrquery" + QM_ELAPSED);
+    prevSolrTime = prevSolrTime == null ? 0 : prevSolrTime;
```

Comment:
❓ What is this for?  I don't see anywhere in this code where this gets set previously so wouldn't this always be null/0?  The responseProps object is created as a new map earlier in this method and that particular key is not added anywhere else.

Reply:
Also, where is `solrQueries` ever being set such that it wouldn't always be null/0?